blockchain/validate.go

```diff
@@ -1196,6 +1196,8 @@ func (b *BlockChain) checkConnectBlock(node *blockNode, block *btcutil.Block, vi
 		}
 	}
 
+	scriptFlags |= txscript.ScriptStrictMultiSig
```

Comment:
This should be gated behind a BIP9 check.  I'm sure it's done in a later commit, but it probably shouldn't be in this one as it would constitute a hard fork as is.